the-collab-lab/tcl-44-smart-shopping-list
# Bugfix - Prevent checkbox showing over navbar

## Description

This PR fixes the issue of the checkbox showing over the navbar

- I added the `z-40` class to the `nav` in `Navbar.js` to fix the bug

## Related Issue

Close #39

## Acceptance Criteria

The checkbox should be under the navbar and not on top of it. 


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|   ✓| :bug: Bug fix              |
|  | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before


![bug-fix-before](https://user-images.githubusercontent.com/84801660/169864434-6bae5ded-f24b-4c39-9cad-d042903aef71.JPG)


### After

![bug-fix-after](https://user-images.githubusercontent.com/84801660/169865065-4e0d5d60-af9d-4974-ad8f-3eefd67a32c4.JPG)



## Testing Steps / QA Criteria

•	From your terminal, pull down this branch with git pull origin `bugfix-prevent-checkbox-showing-over-navbar` and check that branch out with git checkout `bugfix-prevent-checkbox-showing-over-navbar` .
- 	Then npm start to launch the app.
